src/codegen/statements.rs

```diff
@@ -1362,22 +1363,22 @@ impl LoopScopes {
         LoopScopes(LinkedList::new())
     }
 
-    fn new_scope(&mut self, break_bb: usize, continue_bb: usize) {
+    pub(crate) fn new_scope(&mut self, break_bb: usize, continue_bb: usize) {
```

Comment:
`pub(super)` ?

Reply:
What's the advantage of `pub(super)` here? I understand that this function need to be visible in the current create, so that I can use it dealing with yul statements.

Reply:
@seanyoung

Reply:
`pub(crate)` means that the function is only available in the `codegen` crate.

Reply:
`codegen` isn't a crate, it's a module.

`solang` is the crate.